webapp/locales/pt.json

```diff
@@ -39,57 +151,190 @@
     "follow": "Seguir",
     "followers": "Seguidores",
     "following": "Seguindo",
-    "shouted": "Aclamou",
+    "shouted": "Recomendou",
     "commented": "Comentou",
-    "userAnonym": "Anonymous"
+    "userAnonym": "Anônimo",
+    "socialMedia": "Onde mais posso encontrar",
+    "network": {
+      "title": "Rede",
+      "following": "está seguindo:",
+      "followingNobody": "não segue ninguém.",
+      "followedBy": "é seguido por:",
+      "followedByNobody": "não é seguido por ninguém.",
+      "andMore": "e {number} mais …"
+    },
+    "invites": {
+      "title": "Convidar alguém para Human Connection!",
+      "description": "Digite o endereço de e-mail para o convite.",
+      "emailPlaceholder": "E-mail para convidar"
+    }
+  },
+  "notifications": {
+    "reason": {
+      "mentioned_in_post": "Mencionou você em uma publicação …",
+      "mentioned_in_comment": "Mencionou você em um comentário …",
+      "commented_on_post": "Comentou na sua publicação …"
+    },
+    "comment": "Comentário"
+  },
+  "search": {
+    "placeholder": "Pesquisar",
+    "hint": "O que você está pesquisando??",
+    "failed": "Nada foi encontrado"
   },
   "settings": {
     "name": "Configurações",
     "data": {
       "name": "Seus dados",
-      "labelName": "Seu nome",
-      "namePlaceholder": "Anonymous",
-      "labelCity": "Sua cidade ou estado",
-      "labelBio": "Sobre você"
+      "labelName": "Seu Nome",
+      "labelSlug": "Seu nome de usuário exclusivo",
+      "namePlaceholder": "Femanon Funny",
+      "labelCity": "Sua cidade ou região",
+      "labelBio": "Sobre você",
+      "success": "Seus dados foram atualizados com sucesso!"
+    },
+    "email": {
+      "validation": {
+        "same-email": "Este é o seu endereço de e-mail atual"
+      },
+      "name": "Seu email",
+      "labelEmail": "Alterar o seu endereço de e-mail",
+      "labelNewEmail": "Novo endereço de e-mail",
+      "labelNonce": "Digite o seu código",
+      "success": "Um novo endereço de e-mail foi registrado.",
+      "submitted": "Um e-mail para verificar o seu endereço foi enviado para <b>{email}</b>.",
+      "change-successful": "O seu endereço de e-mail foi alterado com sucesso.",
+      "verification-error": {
+        "message": "O seu e-mail não pode ser alterado.",
+        "explanation": "Isto pode ter diferentes causas:",
+        "reason": {
+          "invalid-nonce": "O código de confirmação esta inválido?",
+          "no-email-request": "Você tem certeza de que solicitou uma alteração no seu endereço de e-mail?"
+        },
+        "support": "Se o problema persistir, por favor contacte-nos por e-mail"
+      }
+    },
+    "validation": {
+      "slug": {
+        "regex": "Os caracteres permitidos são apenas letras minúsculas, números, sublinhados e hífens.",
+        "alreadyTaken": "Este nome de usuário já está registrado."
+      }
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-11-04T19:49:24Z" title="Monday, November 4th 2019, 8:49:24 pm +01:00">Nov 4, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -41,193 +136,733 @@
     "following": "Seguindo",
     "shouted": "Aclamou",
     "commented": "Comentou",
-    "userAnonym": "Anonymous"
+    "userAnonym": "Anonymous",
+    "socialMedia": "Where else can I find",
+    "network": {
+      "title": "Network",
+      "following": "is following:",
+      "followingNobody": "follows nobody.",
+      "followedBy": "is followed by:",
+      "followedByNobody": "is not followed by anyone.",
+      "andMore": "and {number} more …"
+    },
+    "invites": {
+      "title": "Invite somebody to Human Connection!",
+      "description": "Enter thier email address for invitation.",
+      "emailPlaceholder": "Email to invite"
+    }
+  },
+  "notifications": {
+    "reason": {
+      "mentioned_in_post": "Mentioned you in a post …",
+      "mentioned_in_comment": "Mentioned you in a comment …",
+      "commented_on_post": "Commented on your post …"
+    },
+    "comment": "Comment"
+  },
+  "search": {
+    "placeholder": "Search",
+    "hint": "What are you searching for?",
+    "failed": "Nothing found"
   },
   "settings": {
-    "name": "Configurações",
+    "name": "Settings",
     "data": {
-      "name": "Seus dados",
-      "labelName": "Seu nome",
-      "namePlaceholder": "Anonymous",
-      "labelCity": "Sua cidade ou estado",
-      "labelBio": "Sobre você"
+      "name": "Your data",
+      "labelName": "Your Name",
+      "labelSlug": "Your unique user name",
+      "namePlaceholder": "Femanon Funny",
+      "labelCity": "Your City or Region",
+      "labelBio": "About You",
+      "success": "Your data was successfully updated!"
+    },
+    "email": {
+      "validation": {
+        "same-email": "This is your current email address"
+      },
+      "name": "Your email",
+      "labelEmail": "Change your email address",
+      "labelNewEmail": "New email Address",
+      "labelNonce": "Enter your code",
+      "success": "A new email address has been registered.",
+      "submitted": "An email to verify your address has been sent to <b>{email}</b>.",
+      "change-successful": "Your email address has been changed successfully.",
+      "verification-error": {
+        "message": "Your email could not be changed.",
+        "explanation": "This can have different causes:",
+        "reason": {
+          "invalid-nonce": "Is the confirmation code invalid?",
+          "no-email-request": "Are you certain that you requested a change of your email address?"
+        },
+        "support": "If the problem persists, please contact us by email at"
+      }
+    },
+    "validation": {
+      "slug": {
+        "regex": "Allowed characters are only lowercase letters, numbers, underscores and hyphens.",
+        "alreadyTaken": "This user name is already taken."
+      }
```


This seems to be English, not Portuguese. :thinking:

Reply:
> [<img alt="ppelegrin" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ppelegrin) **Authored by [ppelegrin](https://github.com/ppelegrin)**
_<time datetime="2019-11-04T19:55:29Z" title="Monday, November 4th 2019, 8:55:29 pm +01:00">Nov 4, 2019</time>_

---


I'm still working on it ;)